### PR TITLE
Jump to the conversation end when sending the message

### PR DESCRIPTION
**Pull request by mikeger**
### Issues

In case we see the message from the current user appearing in the message window, the table view is scrolling to the very end of the conversation.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationMessageWindowTableViewAdapter+DataSource.swift [hunk 1]
```diff
@@ -46,10 +52,14 @@ extension ConversationMessageWindowTableViewAdapter: ZMConversationMessageWindow
         
         let isLoadingInitialContent = messageWindow.messages.count == changeInfo.insertedIndexes.count && changeInfo.deletedIndexes.count == 0
         let isExpandingMessageWindow = changeInfo.insertedIndexes.count > 0 && changeInfo.insertedIndexes.last == messageWindow.messages.count - 1
-        
+        var shouldJumpToTheConversationEnd = false
         stopAudioPlayer(forDeletedMessages: changeInfo.deletedObjects)
         
-        if isLoadingInitialContent || (isExpandingMessageWindow && changeInfo.deletedIndexes.count == 0) || changeInfo.needsReload {
+        if changeInfo.insertedMessages.any({ $0.sender?.isSelfUser ?? false }) {
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationMessageWindowTableViewAdapter+DataSource.swift]: Won't this jump to the end of you send a message from another device as well?

Comment (mikeger) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationMessageWindowTableViewAdapter+DataSource.swift]: This is bothering me as well, but not 100% percent sure it's something that can be bad for users.

Comment (typfel) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationMessageWindowTableViewAdapter+DataSource.swift]: Yes, maybe it's ok. What's the alternatives, trigger the scroll when inserting the message?

**Comment by mikeger** [comment]
@typfel updated with the nicer solution, which can be condensed to 1 line change:
https://github.com/wireapp/wire-ios/pull/2876/files#diff-d0adf7620aab7082451935f800968beaR748

Basically when user clicks the send button we are bringing the input field into focus in order to finish the autocomplete. It was just broken because the call to scroll to the bottom with parameter animated true was somehow not happening to do anything.

**Comment by mikeger** [comment]
@typfel please review when you have time.

**Pull request merged**